libs/qec/python/bindings/py_code.cpp

```diff
@@ -310,6 +330,17 @@ void bindCode(py::module &mod) {
   qecmod.def("get_available_codes", &get_available_codes,
              "Get a list of all available quantum error correction codes");
 
+  qecmod.def(
+      "get_all_available_codes",
```

Comment:
In reference to the other comment about `get_all_available_codes`, what I am essentially recommending is deleting lines 330-331 and replacing line 334 as shown here:
```suggestion
      "get_available_codes",
```